lib/domain/model/ongoing_call.dart

```diff
@@ -2154,7 +2162,56 @@ class OngoingCall {
             devices.firstWhereOrNull(
               (e) => e.deviceId() == track.getTrack().deviceId(),
             );
+
+        await _addNoiseSuppression(track);
+      }
+    }
+  }
+
+  /// Adds [NoiseSuppressionLevel] to [LocalMediaTrack].
+  ///
+  /// Available only on Desktop now.
+  Future<void> _addNoiseSuppression(LocalMediaTrack track) async {
+    Log.debug('_addNoiseSuppression($track)', '$runtimeType');
+
+    if (!track.isAudioProcessingAvailable()) {
+      Log.debug(
+        '❌ Audio processing not available for this track $track',
+        '$runtimeType',
+      );
+      return;
+    }
```

Comment:
I guess this should be checked only if noise suppression is enabled and the current platform is supported. Otherwise I don't see any reason to check it.